src/sentry/constants.py

```diff
@@ -84,7 +84,7 @@ def get_all_languages():
     'subscribe', 'enterprise', 'about', 'jobs', 'thanks', 'guide',
     'privacy', 'security', 'terms', 'from', 'sponsorship', 'for',
     'at', 'platforms', 'branding', 'vs', 'answers', '_admin',
-    'support', 'contact',
+    'support', 'contact', 'onboarding',
```

Comment:
Have we confirmed that we can freely claim this url without impacting customers?